docs/dev/checks/README.md

```diff
@@ -146,13 +155,50 @@ dev/dist
 └── README.md
 ```
 
+This is sufficient to have the Agent correctly discover both the code and the
+configuration, but in order to actually run this, we must [discuss
+dependencies](https://xkcd.com/1987/)
+
+### Python Check Dependencies
+When `hello_world.py` runs, it loads in the `AgentCheck` class which provides
+some default functionality (eg, the `self.gauge` method)
+
+In order for python to find this package, we must do two things:
+1. Install the `integrations-core` dependencies
+2. Set the `PYTHONPATH` env var correctly to find those dependencies
+
+`datadog-checks-base` must be installed as a pre-req, and you can install via a
+`--user` install or a `virtualenv` install, as long as you set your `PYTHONPATH`
+correctly.
+
+#### Example for virtualenv
+(see also the notes in [../agent_dev_env.md](../agent_dev_env.md)):
+
+1. `python3 -m pip install virtualenv`
+1. `virtualenv $GOPATH/src/github.com/DataDog/datadog-agent/venv`
+1. Activate the virtualenv (OS-dependent)
+1. `python3 -m pip install ~/dev/integrations-core/datadog_checks_base[deps]`
+1. `PYTHONPATH="$PWD/venv/lib/python3.10/site-packages:$PYTHONPATH" inv agent.run`
+
+
+#### Example for user install
+1. `python3 -m pip install --user ~/dev/integrations-core/datadog_checks_base[deps]`
```

Comment:
```suggestion
1. `python3 -m pip install --user ~/dev/integrations-core/'datadog_checks_base[deps]'`
```

I think it needs to be formatted like this. (did not work as-is on my machine in ZSH)